docs/book/v3/features/template/twig.md

```diff
@@ -33,13 +33,11 @@ Alternately, you can instantiate and configure the engine yourself, and pass it
 to the `Mezzio\Twig\TwigRenderer` constructor:
 
 ```php
-use Twig_Environment;
-use Twig_Loader_Array;
 use Mezzio\Twig\TwigRenderer;
 
 // Create the engine instance:
-$loader = new Twig_Loader_Array(include 'config/templates.php');
-$twig = new Twig_Environment($loader);
+$loader = new Twig\Loader\ArrayLoader(include 'config/templates.php');
+$twig = new Twig\Environment($loader);
```

Comment:
Since these weren't imported, perhaps `\Twig\Environment` and such?

Reply:
This is a stand-alone example, so there is no need to import anything if it is only used once. And the first backslash is also not necessary.

Reply:
Hmm, it's a bit inconsistent between `use` and no `use` then :thinking:

Reply:
I'm happy to stick with use for the sake of consistency.